llvm/test/CodeGen/ARM/ssp-data-layout.ll

```diff
@@ -443,4 +443,4 @@ declare void @end_struct_large_nonchar()
 declare signext i16 @get_struct_small_nonchar()
 declare void @end_struct_small_nonchar()
 
-declare void @takes_all(i64, i16, ptr byval(%struct.struct_large_nonchar) align 8, i32, ptr, ptr, ptr, ptr, ptr, i32, i32, i32)
+declare void @takes_all(i64, i16, ptr byval(%struct.struct_large_nonchar) align 4, i32, ptr, ptr, ptr, ptr, ptr, i32, i32, i32)
```

Comment:
Can you explain this change?

Reply:
In the test they always override callsite param with `align 4`. After this change it switches to 8 which causes ALOT of diffs in the stack allocations. Felt like a mistake when writing the test to have the two diverge.

But probably should add an ARM maintainer to this to be sure.

Reply:
@davemgreen this okay?

Reply:
Sorry, I didn't see this until now. This test is very old now but considering what it does, yes this sounds OK.